vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+
+### The GitHub Packages NuGet cache provider
+
+[GitHub Packages](https://docs.github.com/packages/learn-github-packages/introduction-to-github-packages) allows a repository to publish binary artifacts for public or private use. In addition to hosting storage for published packages, GitHub Packages supports a [variety of package management tools](https://docs.github.com/packages/learn-github-packages/introduction-to-github-packages#supported-clients-and-formats) by acting as a package registry. vcpkg can use the NuGet registry interface to GitHub Packages as a cache for vcpkg binary artifacts, by using the [`nuget` binary source provider](./users/binarycaching.md#nuget).
+
+## The GitHub dependency graph
+
+[!INCLUDE [experimental](../includes/experimental.md)]
+
+> [!TIP]
+> We are actively developing this feature and would like to hear your feedback. Let us know if you have any thoughts about the current functionality or any desired functionality by emailing [vcpkg@microsoft.com](mailto:vcpkg@microsoft.com) or by filing an issue in [vcpkg-docs](https://github.com/microsoft/vcpkg-docs/issues). If you have feedback on any of the GitHub features that this enables, let us know that too and we'll make sure it gets to the right folks at GitHub.
+
+### About the GitHub dependency graph
+
+The GitHub [dependency graph](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) stores the set of dependencies for a repository. Beyond being able to visualize a repository's dependencies, GitHub builds several useful features on top of this data, including [dependency review](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review) and [Dependabot alerts](https://docs.github.com/code-security/dependabot/dependabot-alerts/about-dependabot-alerts).
+
+### vcpkg integration with the GitHub dependency graph
+
+vcpkg has experimental support for populating the GitHub dependency graph from within a GitHub Actions workflow. To enable this feature, make the following changes in your workflow file:
+
+* Set the `VCPKG_FEATURE_FLAGS` environment variable to include the value `dependencygraph`. 
+* Set the `GITHUB_TOKEN` environment variable to the value `${{ secrets.GITHUB_TOKEN }}`.
+* Give GitHub Actions write access to the contents of your repository by including the following permissions block. This is required to write the dependency graph metadata to your repository. This __will not__ add any commits to your repository or make any other modification to the contents of your repository.
```

Comment:
Where does this permissions block change go?

Reply:
There is pretty much only one place it can go in the file, and there is a concrete example of where that is in the example below. For our blog post, I also plan to link to a live example, but I'm opting not to link to that in the official documentation (at least not yet anyways).